projects/knora-ui/README.md

```diff
@@ -1,25 +1,25 @@
-# KnoraUi
+# @knora/ui
 
 Reusable Angular elements for Angular applications based on knora-api.
 
 This library was generated with [Angular CLI](https://github.com/angular/angular-cli) version 8.2.14.
 
 ## Code scaffolding
 
-Run `ng generate component component-name --project knora-ui` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module --project knora-ui`.
-> Note: Don't forget to add `--project knora-ui` or else it will be added to the default project in your `angular.json` file. 
+Run `ng generate component component-name --project @knora/ui` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module --project @knora/ui`.
+> Note: Don't forget to add `--project @knora/ui` or else it will be added to the default project in your `angular.json` file.
 
 ## Build
 
-Run `ng build knora-ui` to build the project. The build artifacts will be stored in the `dist/` directory.
+Run `ng build @knora/ui` to build the project. The build artifacts will be stored in the `dist/` directory.
 
 ## Publishing
 
-After building your library with `ng build knora-ui`, go to the dist folder `cd dist/knora-ui` and run `npm publish`.
+After building your library with `ng build @knora/ui`, go to the dist folder `cd dist/knora/ui` and run `npm publish`.
 
 ## Running unit tests
 
-Run `ng test knora-ui` to execute the unit tests via [Karma](https://karma-runner.github.io).
```

Comment:
We have a script `npm run test-lib` or `npm run test-lib-local`.

Reply:
This should be part of other PR and user story e.g. https://dasch.myjetbrains.com/youtrack/issue/DSP-199